### PR TITLE
Fixed a couple of tiny bugs

### PR DESCRIPTION
**Pull request by davivel**

### CODE REVIEW

**Comment by masensio** [comment]
Reviewed!! Ready for QA

**Comment by davivel** [comment]
Last update. I promise.

Please, @masensio , review again.

File: src/com/owncloud/android/providers/FileContentProvider.java [hunk 1]
```diff
@@ -178,6 +178,7 @@ private int delete(SQLiteDatabase db, Uri uri, String where, String[] whereArgs)
             if (c != null && c.moveToFirst()) {
                 remoteId = c.getString(c.getColumnIndex(ProviderTableMeta.FILE_REMOTE_ID));
                 //ThumbnailsCacheManager.removeFileFromCache(remoteId);
+                c.close();
```
Comment (masensio) [inline src/com/owncloud/android/providers/FileContentProvider.java]: Close cursors is a good idea!

File: src/com/owncloud/android/utils/FileStorageUtils.java [hunk 2]
```diff
@@ -46,12 +46,12 @@
...
  */
 public class FileStorageUtils {
-    public static Integer mSortOrder;
-    public static Boolean mSortAscending;
     public static final Integer SORT_NAME = 0;
     public static final Integer SORT_DATE = 1;
     public static final Integer SORT_SIZE = 2;
-  
+    public static Integer mSortOrder = SORT_NAME;
+    public static Boolean mSortAscending = true;
```
Comment (masensio) [inline src/com/owncloud/android/utils/FileStorageUtils.java]: +1 init variables

**Comment by masensio** [comment]
Ready for QA again!

**Comment by Dianafg76** [comment]
I tested and it is working OK

**Pull request merged**